cmd/manifest/manifest.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
This is grim.  Do we really need to solve this problem here?  End users can just edit their manifest yaml.  Scale testing can create its own manifest yaml and modify it using its own separate codebase

Reply:
I don't care where this lives but it's something that can evolve to a e2e library.

Reply:
removed for now, didnt add much indeed